pages/pets/show.tsx

```diff
@@ -75,13 +75,13 @@ const GamesShow:React.FC<GamesShowProps> = () => {
   
   return (
     <div>
-      <p>Got a game, baby!</p>
-      {game ? 
-        <p>{game[0].name}</p>
+      <p>Got a pet, baby!</p>
+      {pet ? 
+        <p>{pet[0].name}</p>
```

Comment:
Once we type the pet state above, this will error. Just removing the `[0]` should fix that.